models/aeif_psc_delta.cpp

```diff
@@ -271,7 +273,9 @@ nest::aeif_psc_delta::State_::set( const DictionaryDatum& d, const Parameters_&,
   updateValueParam< double >( d, names::w, y_[ W ], node );
 }
 
-nest::aeif_psc_delta::Buffers_::Buffers_( aeif_psc_delta& n )
+
+nest::aeif_psc_delta::Buffers_::Buffers_( // @suppress("Class members should be properly initialized")
```

Comment:
Is this ``// @suppress`` Eclipse syntax? Is it desirable to have these comments in upstream?

This also raises the question why the directive is here (what are we suppressing and why)?

Reply:
Fixed, I removed this again, it is too Eclipse-specific. The warning is about missing initialisation, but that is handled in `init_buffers()`, and this again explained in a constructor comment.